packages/core/pattern.mjs

```diff
@@ -340,9 +340,9 @@ export class Pattern {
    * silence
    * @noAutocomplete
    */
-  queryArc(begin, end) {
+  queryArc(begin, end, controls = {}) {
     try {
-      return this.query(new State(new TimeSpan(begin, end)));
+      return this.query(new State(new TimeSpan(begin, end, controls)));
```

Comment:
shouldn't controls be passed to State and not TimeSpan?

Reply:
Oops yes, fixed.. It actually works now!